decision-making-processes.md

```diff
@@ -1,35 +1,157 @@
 
-Types of decisions include: 
-* Who is on the Solid team? 
-* How to make a change to the Solid spec? 
-* How to start a Solid Project? 
- 
-Possible candidates for the Decision Panel, that could be adapted according to the type of decision include: 
+The Solid Team assume [defined roles](https://github.com/solid/information/blob/master/solid-team.md) required to achieve the Solid project's goals, especially those that require a high level of trust. The defined [roles and responsibilities](https://github.com/solid/information/blob/master/solid-team.md) as well as indiviudals apointed were defined by open suggestions by anyone on a Github pull request which were ultimately approved by the Solid Leader, Tim Berners-Lee. 
+
+## Mandate 
+The Solid team shall work to: 
+* Maintain the quality and stability of Solid 
+* Make contributing as accessible, inclusive, and sustainable as possible 
+* Establish appropriate decision-making processes
+* Seek consensus among those building on Solid and using Solid before acting in a formal capacity 
+* Act as a court of final appeal for decisions where all other methods have failed
+* Coordinating the definition of the Solid values
+* Coordinating the definition of the Solid specifications including from technical, legal, and design perspectives
+* Providing examples of the Solid specification under open source licensing
+* Promoting awareness of Solid
+* Coordinating the several parties implementing the Solid specification 
+* Coordinating the development of a Solid Test suite to automate the checking of compliance of solutions to the Solid specifications
+* Providing an updated list of Solid solutions
+
+## Powers 
+The Solid Team has broad authority to make decisions about Solid. For example, they can: 
+* Formally accept or reject suggestions (usually in the form of a GitHub pull request or issue) 
+* Enforce or update the Solid project's code of conduct 
+* Manage Solid assets and infrastructure, including the Solid Github organisation and repositories, the bug tracker, mailing lists, conversation channels etc.
+
+However, the Solid Team cannot modify this decision making process or affect the membership of the Solid Panel, except via the mechanisms specified in this decision making process document. 
+
+The Solid Team should look for ways to use these powers as little as possible. Instead of voting it's better to seek consensus. Instead of ruling on individuals it's better to define standard processes for decision making. It's better to establish a Code of Conduct committee than to rule on individual cases, etc. 
+
+To use its powers, the Solid Team votes. The Solid leader can always veto the voting outcome. Every Solid Team member  must either vote or explicitly abstain. Members with conflicts of interest on a particular vote must abstain. Passing requires a strict majority of non-abstaining Solid Team members.
+
+Whenever possible, the Solid Team's deliberations and votes shall be held in public.
```

Comment:
Cases where this is not possible are…?

Reply:
You mean the Solid leader veto or that the deliberations of the Solid Team being public? 

Do you have cases in mind?

Reply:
It says "whenever possible", so I'd want to make that explicit, or maybe remove if no cases exist. I don't see any (unless urgent security bugs).